packages/ember-metal/lib/expand_properties.js

```diff
@@ -5,7 +5,8 @@ import { assert } from 'ember-debug';
 @submodule ember-metal
 */
 
-var END_WITH_EACH_REGEX = /\.@each$/;
+var END_WITH_EACH_REGEX = /\.@each$/,
+    NESTED_BRACES = /\{[^}{]*\{|\}[^}{]*\}|\{[^}]*$/g;
```

Comment:
we should likely in-line this into 46, and leave a comment. That way when the assert is stripped the RegExp is aswell.